vtrace/platforms/linux.py

```diff
@@ -413,6 +413,12 @@ def platformAttach(self, pid):
             raise Exception("PT_ATTACH failed!")
         self.setMeta("ExeName", self._findExe(pid))
 
+    def _LibraryLoadHack(self):
+        # drop special breakpoint at ld._dl_catch_exception
+        import vtrace.breakpoints as v_bp
+        bp = v_bp.PosixLibLoadHookBreakpoint('ld._dl_catch_exception')
+        self.addBreakpoint(bp)
+
```

Comment:
in the next file (`vtrace/platforms/posix.py`) we see a call to _LibraryLoadHack, but since i'm only certain of this ld._dl_catch_exception behavior on Linux, we implement that part here.